financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -148,6 +159,82 @@ contract("CreatePriceFeed.js", function(accounts) {
+    // Given the collateral token is 0x1, the , it should always come first, meaning the config should always be inverted.
+    const collateralTokenAddress = "0x0000000000000000000000000000000000000001";
+
+    const constructorParams = {
+      expirationTimestamp: (Math.round(Date.now() / 1000) + 1000).toString(),
+      withdrawalLiveness: "1000",
+      collateralAddress: collateralTokenAddress,
+      finderAddress: Finder.address,
+      tokenFactoryAddress: TokenFactory.address,
+      priceFeedIdentifier: web3.utils.utf8ToHex("ETH/BTC"),
+      syntheticName: "Test UMA Token",
+      syntheticSymbol: "UMATEST",
+      liquidationLiveness: "1000",
+      collateralRequirement: { rawValue: toWei("1.5") },
+      disputeBondPct: { rawValue: toWei("0.1") },
+      sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
+      disputerDisputeRewardPct: { rawValue: toWei("0.1") },
+      minSponsorTokens: { rawValue: toWei("1") },
+      timerAddress: Timer.address
+    };
+
+    const emp = await ExpiringMultiParty.new(constructorParams);
+
+    const getIdBackup = web3.eth.net.getId;
+
+    // Modify web3 to say the chain id is mainnet temporarily.
+    web3.eth.net.getId = async () => 1;
+
+    const twapLength = 100;
+    const priceFeed = await createUniswapPriceFeedForEmp(logger, web3, networker, getTime, emp.address, { twapLength });
+
+    // Cannot test for the uniswap address since that depends on the synthetic token address, which is generated in a non-hermetic way.
+    // Price should always be inverted since the collateralTokenAddress is 0x1.
+    assert.isTrue(priceFeed.invertPrice);
+
+    // Config override should be passed through.
+    assert.equal(priceFeed.twapLength, twapLength);
+
+    // Reset getId method.
+    web3.eth.net.getId = getIdBackup;
+  });
+
+  it("Uniswap address not found", async function() {
+    const collateralToken = await Token.new("UMA", "UMA", 18, { from: accounts[0] });
+
+    const constructorParams = {
+      expirationTimestamp: (Math.round(Date.now() / 1000) + 1000).toString(),
+      withdrawalLiveness: "1000",
+      collateralAddress: collateralToken.address,
+      finderAddress: Finder.address,
+      tokenFactoryAddress: TokenFactory.address,
+      priceFeedIdentifier: web3.utils.utf8ToHex("ETH/BTC"),
+      syntheticName: "Test UMA Token",
+      syntheticSymbol: "UMATEST",
+      liquidationLiveness: "1000",
+      collateralRequirement: { rawValue: toWei("1.5") },
+      disputeBondPct: { rawValue: toWei("0.1") },
+      sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
+      disputerDisputeRewardPct: { rawValue: toWei("0.1") },
+      minSponsorTokens: { rawValue: toWei("1") },
+      timerAddress: Timer.address
+    };
+
+    const emp = await ExpiringMultiParty.new(constructorParams);
+
+    let didThrow = false;
+    try {
+      const priceFeed = await createUniswapPriceFeedForEmp(logger, web3, networker, getTime, emp.address);
```

Comment:
To clarify: why is this uniswap address not found? because there is no uniswap contract on the network?

Reply:
Correct. I'll add a comment.

Reply:
DOne.